scripts/check_sss_conflicts.py

```
@@ -0,0 +1,274 @@
+        if 'config.yaml' not in files:
+            continue
+
+        config_path = os.path.join(root, 'config.yaml')
+
+        if args.verbose:
+            print(f"Processing: {config_path}")
+
+        config = parse_config(config_path)
+        if not config:
+            if args.verbose:
+                print(f"  Skipped (not a SelectorSyncSet)")
```

Ruff (0.15.18)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)